tracee-ebpf/tracee/containers.go

```diff
@@ -178,7 +182,9 @@ func (c *Containers) CgroupUpdate(cgroupId uint64, path string) (CgroupInfo, err
 		info.Runtime = runtime
 	}
 
```

Comment:
Note that here we always fall in the statement that updates the `c.cgroups `map, even if `info.ContainerId` and `info.Runtime` is not initialized. Is it intentional @yanivagman ?